### PR TITLE
Kredx5

### PR DESCRIPTION
**Pull request by kulkarnikarthik**

### CODE REVIEW

File: src/test/java/base/BasePage.java [hunk 1]
```diff
@@ -45,4 +46,30 @@ protected void waitForLoaderToDisappear() {
             e.printStackTrace();
         }
     }
+
+    public String mobilenumber(){
```
Comment (njchandu) [inline src/test/java/base/BasePage.java]: rename this to generateRandomMobileNumber()

File: src/test/java/base/BasePage.java [hunk 2]
```diff
@@ -45,4 +46,30 @@ protected void waitForLoaderToDisappear() {
...
+        String num = "710040";
+
+        Random random = new Random();
+        String endingNum = String.format("%04d", random.nextInt(10000));
+        return num + endingNum;
+    }
+
+    public String ramdomPassword(){
```
Comment (njchandu) [inline src/test/java/base/BasePage.java]: rename this to generateRandomPassword()

File: src/test/java/base/BasePage.java [hunk 3]
```diff
@@ -45,4 +46,30 @@ protected void waitForLoaderToDisappear() {
...
+        String alphaNumerics = "qwertyuiopasdfghjklzxcvbnm1234567890";
+        String password = "";
+        for (int i = 0; i < 8; i++) {
+            password += alphaNumerics.charAt((int) (Math.random() * alphaNumerics.length()));
+        }
+        return password;
+    }
+
+    public String randomName(){
```
Comment (njchandu) [inline src/test/java/base/BasePage.java]: rename this to generateRandomName()

File: src/test/java/pages/DashboardPage.java [hunk 4]
```diff
@@ -12,11 +15,43 @@
     @FindBy(xpath = "//a[@href='/financier/dashboard/deals']")
     WebElement dealsLink;
 
+   // @FindBy(xpath = "//span[@class = 'icon-dashboard']")
```
Comment (njchandu) [inline src/test/java/pages/DashboardPage.java]: remove the commented portion

File: src/test/java/pages/DashboardPage.java [hunk 5]
```diff
@@ -12,11 +15,43 @@
...
+    }
+
+    public DashboardPage logoutUser(){
+
+        clickOnProfileDropdown.click();
+        logoutButton.click();
+        return this;
+
+    }
+
```
Comment (njchandu) [inline src/test/java/pages/DashboardPage.java]: lots of extra lines, between code. prefer to keep it clean :)

File: src/test/java/pages/DashboardPage.java [hunk 6]
```diff
@@ -12,11 +15,43 @@
...
     public DashboardPage(WebDriver driver) {
         super(driver);
     }
 
     public void clickDeals() {
         dealsLink.click();
     }
+
+    public String confirmOnDashboardPage(){
+
```
Comment (njchandu) [inline src/test/java/pages/DashboardPage.java]: avoid extra line inside method

File: src/test/java/pages/LoginPage.java [hunk 7]
```diff
@@ -33,4 +48,61 @@ public LoginPage enterPassword(String password) {
...
+        }
+    }
+
+    public LoginPage goToSignUpAsInvestorPage(){
+
+        WebDriverWait wait = new WebDriverWait(driver,3000);
+        wait.until(ExpectedConditions.elementToBeClickable(signUpAsInvestor));
+        signUpAsInvestor.click();
+
+        try {
```
Comment (njchandu) [inline src/test/java/pages/LoginPage.java]: preferably get rid of sleeps

File: src/test/resources/test-suite/create_flow.xml [hunk 8]
```diff
@@ -3,7 +3,7 @@
 <suite name="Create Flow">
     <test name="kredx-test-automation">
         <classes>
-            <class name="tests.CreateFlowTest"/>
+            <class name="tests.LoginFlowTest"/>
```
Comment (njchandu) [inline src/test/resources/test-suite/create_flow.xml:6]: Add SignupTest here

File: src/test/java/base/BasePage.java (hunk 1 above)
Comment (kulkarnikarthik) [reply src/test/java/base/BasePage.java]: done

File: src/test/java/base/BasePage.java (hunk 2 above)
Comment (kulkarnikarthik) [reply src/test/java/base/BasePage.java]: done

File: src/test/java/base/BasePage.java (hunk 3 above)
Comment (kulkarnikarthik) [reply src/test/java/base/BasePage.java]: done

File: src/test/java/pages/DashboardPage.java (hunk 4 above)
Comment (kulkarnikarthik) [reply src/test/java/pages/DashboardPage.java]: done

File: src/test/java/pages/DashboardPage.java (hunk 6 above)
Comment (kulkarnikarthik) [reply src/test/java/pages/DashboardPage.java]: done for all methods

File: src/test/java/pages/DashboardPage.java (hunk 5 above)
Comment (kulkarnikarthik) [reply src/test/java/pages/DashboardPage.java]: done

File: src/test/resources/test-suite/create_flow.xml (hunk 8 above)
Comment (kulkarnikarthik) [reply src/test/resources/test-suite/create_flow.xml:6]: done

File: src/test/java/pages/LoginPage.java (hunk 7 above)
Comment (kulkarnikarthik) [reply src/test/java/pages/LoginPage.java]: done

File: src/test/resources/test-suite/create_flow.xml (hunk 8 above)
Comment (njchandu) [reply src/test/resources/test-suite/create_flow.xml:6]: create a regression-test-suite.xml file and add both test suite files ( <suite-files></suite-files>) and update the name here: https://github.com/kulkarnikarthik/kredx-test-infra/blob/86fcfc7d4dc4afec911b8ad8035c6afe3e3e9810/pom.xml#L105

**Review by njchandu — CHANGES_REQUESTED** [review]

File: src/test/resources/test-suite/create_flow.xml (hunk 8 above)
Comment (kulkarnikarthik) [reply src/test/resources/test-suite/create_flow.xml:6]: done

**Comment by njchandu** [comment]
:shipit:

**Pull request merged**